cdap-app-templates/cdap-etl/cdap-etl-proto/src/main/java/io/cdap/cdap/etl/proto/v2/ETLConfig.java

```diff
@@ -265,6 +270,7 @@ protected Builder() {
       this.stageLoggingEnabled = true;
       this.processTimingEnabled = true;
       this.properties = new HashMap<>();
+      this.comments = new ArrayList<>();
```

Comment:
what is this comments? Do we use it anywhere? If not, let's not modify it.

Reply:
Yes we can add comments to a pipeline in pipeline studio